EPR/Library/tests/Makefile

```diff
@@ -2,29 +2,36 @@
 # this is the EPR/Library/tests Makefile
 #https://stackoverflow.com/a/28236905
 #https://gcc.gnu.org/onlinedocs/gcc/Invoking-G_002b_002b.html
+
 #for debugging information, add -g to this next line or in a recipe, below
-CC = g++ -std=c++17 -Wall -Wextra -Werror -O2 -DSGI32
+CC = g++ -std=c++17 -Wall -Wextra -O2 -DSGI32
```

Comment:
We may have competing makefiles on the main join of all the PRS but I think as long as we pay attention we can avoid issues. The main thing is the compile flags seem to be changing but I think we should be okay. May cause a conflict

Reply:
I think we can go ahead and use all three of the -W flags, because they are "just" warnings.

Reply:
The only reason that I deleted that flag was because I was having issues compiling when I would get warnings from other modules that interact with this code (I believe at the time it was octree). I would recommend that we get rid of that, with the understanding that we're going to fix the warnings as they come up, but they shouldn't break our stuff during the development process